pytest/tests/sanity/network_drop_package.py

```diff
@@ -51,16 +47,20 @@ async def handle(self, msg, fr, to):
         return not drop
 
 
-start_cluster(3, 0, 1, None, [["epoch_length", 500]], {}, Handler)
+if __name__ == '__main__':
+    success = Value('i', 0)
+
+    start_cluster(3, 0, 1, None, [["epoch_length", 500]], {},
+                  partial(Handler, success=success))
```

Comment:
personally I prefer multiline arguments, if you like that too you can achive that with default formatter by adding trailing comma in the argument list